userspace/libsinsp/sinsp.h

```diff
@@ -992,7 +987,7 @@ VISIBILITY_PRIVATE
 
         static inline ppm_event_flags simple_consumer_skip_flags()
```

Comment:
the name as no more sense i will rename it in something like `unused_skip_flags` or something similar